data/services/cdn-route.toml

```diff
@@ -1,3 +1,3 @@
 name = "cdn-route"
-description = "Custom routes with HTTPS certificates using Amazon CloudFront"
+description = "Custom domains, CDN caching, and TLS certificates using Amazon CloudFront"
```

Comment:
Might be good to add a note about this bit in the cg-site README, so that we remember to update both next time we go through this process - up to you.